packages/mds-types/index.ts

```diff
@@ -262,6 +334,23 @@ export const TAXI_STATE_EVENT_MAP = TaxiStatusEventMap({
   stopped: ['trip_stop', 'reservation_stop']
 })
 
+export const TNC_STATE_EVENT_MAP = TncStatusEventMap({
+  available: [
+    'driver_cancellation',
+    'enter_jurisdiction',
+    'passenger_cancellation',
+    'provider_cancellation',
+    'service_start',
+    'trip_end'
+  ],
+  reserved: ['reservation_start', 'comms_restored', 'enter_jurisdiction'],
+  non_operational: ['enter_jurisdiction', 'maintenance', 'service_end'],
+  on_trip: ['trip_start', 'enter_jurisdiction', 'comms_restored', 'trip_resume'],
```

Comment:
Missing the cancellation events?

Reply:
Yup yup, need to add those (probably missing for Taxi as well, they were added very late in the dev cycle)

Reply:
I think my answer to this is the same as the comment above, pretty sure this is good to go